src/main/java/entities/Portfolio.java

```diff
@@ -119,4 +119,13 @@ public void pullStocks(List<Stock> newStocks) {
     public Stock convertStockDSResponse(StockDSResponse dsResponse) {
         return stockFactory.createStock(dsResponse.getSymbol(), dsResponse.getValue(), dsResponse.getQuantity(), dbGateWay);
     }
+
+    public double getNetValue() {
+        double total = 0;
+        for (Stock stock : symbolToStock.values()) {
+            total += stock.getValue();
```

Comment:
getValue() only returns the value of one stock of that type, it doesn't account for having multiple stocks. 

Your function should add stock.getValue() * stock.getQuantity() to total for each stock instead of just stock.getValue() to account for multiple stocks of the same type.